### PR TITLE
Bumping ohai and mixlib-log to fix regression

### PR DESCRIPTION
**Pull request by mwrock**
Regresion reported in https://github.com/chef/mixlib-log/issues/17

### CODE REVIEW

File: kitchen-tests/Gemfile.lock [hunk 1]
```diff
@@ -59,9 +59,8 @@ GEM
     excon (0.51.0)
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.14)
```
Comment (btm) [inline kitchen-tests/Gemfile.lock]: @mwrock I believe this is a recurring bug (when working from a windows box?) and you need to add this line back.

Comment (mwrock) [inline kitchen-tests/Gemfile.lock]: i noticed that. I actually ran `rake dependencies:update` on ubuntu. Was gonna just let it be unless it breaks.

Comment (mwrock) [inline kitchen-tests/Gemfile.lock]: Looking at the whole gemfile.lock I think thats the right thing. windows is the only platform that needs that gem and it kept the mingw version

Comment (btm) [inline kitchen-tests/Gemfile.lock]: Will see if this happens again, this is why I added 2da9c1b20c11ce944835ea818a3c81566af7d739 last time.

https://travis-ci.org/chef/chef/jobs/149830214

Comment (btm) [inline kitchen-tests/Gemfile.lock]: actually, looks like it already happened again: https://travis-ci.org/chef/chef/jobs/151909377

Comment (mwrock) [inline kitchen-tests/Gemfile.lock]: yup. guess we need to look into why that is getting yanked but will add it manually now

**Pull request merged**